services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/Orders/SetAsCancelledDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace EShopOnAbp.OrderingService.Orders
+{
+    public class SetAsCancelledDto
+    {
+        public Guid PaymentRequestId { get; set; }
```

Comment:
I don't think this dto is necessary. We will cancel the order, not the payment request used for the order.
I get that, this is used for refunding. However we should use existing paymentRequestId on the order instead of sending from frontend.